mimic/rest/loadbalancer_api.py

```diff
@@ -18,6 +18,7 @@
 from random import randrange
 
 from mimic.util.helper import invalid_resource, json_dump
+from characteristic import attributes
```

Comment:
Definitely not a blocker for this PR, but we are slowly migrating from [Characteristic](https://warehouse.python.org/project/characteristic/) to [`attrs`](https://warehouse.python.org/project/attrs/), so you should be aware of that for new objects.  (You can think of `attrs` as the next version of Characteristic, with a more compact API.